### PR TITLE
Requesting signoff for Mold Subsystem

### PR DESCRIPTION
**Pull request by docrobbins4**

### CODE REVIEW

File: Documentation/Signoffs/dylanrobbins-mold-signoff.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+![image](https://github.com/jacksonrwoodard/HouseHealthMonitoring/assets/104484972/3401a3b6-74a1-49af-a090-dfe94abc742c)
+
+
+## Constraints
+| No. | Constraints | Origin |
+| --- | ----------- | ------ |
+|  1  | Shall be able to detect temperature ranges between 55&deg; F - 85&deg; F and humidity levels exceeding 50% RH. | Project Team |
+|  2  | Shall be able to communicate through I2C protocol and send data every hour to its local ESP32-H2 transmitter. | Project Team |
+|  3  | Shall be protected and not exposed to harsh environmental conditions to prevent any damage to the sensor. | All External Stakeholders, Ethics, & Team Supervisor |
+|  4  | Shall give precise readings to properly evaluate and determine if mold like conditions are present. | Project Team |
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/dylanrobbins-mold-signoff.md]: this is not specific enough.

**Review by JesseTNRoberts — COMMENTED** [review]
Nothing here about sleep mode. I'm not confident that these are wired correctly to permit communication.

File: Documentation/Signoffs/dylanrobbins-mold-signoff.md (hunk 1 above)
Comment (JesseTNRoberts) [reply Documentation/Signoffs/dylanrobbins-mold-signoff.md]: I have addressed that this is not sufficiently precise.

**Review by JesseTNRoberts — COMMENTED** [review]
Prove that the connections made will function based on datasheets.

**Comment by docrobbins4** [comment]
> Nothing here about sleep mode. I'm not confident that these are wired correctly to permit communication.

The sleep mode is targeted towards the ESP32-H2 on the communication side. I added on to the constraint to be more specific about the sleep mode. I added a reference to the datasheet in the schematic section stating that the ESP32-H2 can support I2C on any GPIO pin.

File: Documentation/Signoffs/dylanrobbins-mold-signoff.md (hunk 1 above)
Comment (docrobbins4) [reply Documentation/Signoffs/dylanrobbins-mold-signoff.md]: I added specific reading ranges to be more precise in the constraint.

**Comment by docrobbins4** [comment]
> Prove that the connections made will function based on datasheets.

I added a reference to the datasheet in the schematic section stating that the ESP32-H2 can support I2C on any GPIO pin.

**Review by JesseTNRoberts — COMMENTED** [review]
Wireless has nothing to do with I2c

**Pull request merged**